cypress/fixtures/connectorTypes2.json

```diff
@@ -0,0 +1,3622 @@
+{
```

Comment:
Adding a new copy to prevent the cy test from failing, I will plan to update the old file, removing this one and also updating the cy tests.

Reply:
> Adding a new copy to prevent the cy test from failing, I will plan to update the old file, removing this one and also updating the cy tests.

That sounds good.  

As an aside I've been wondering as well if eventually we could generate some of these fixtures programatically in the tests since they're defined in the API and eventually move away from static fixtures.

Reply:
sure, it would be good, we may have to look. Do you have any references or lib in mind?

Reply:
@indraraj something that's like https://json-generator.com/ but as a library.  I've been thinking that our integration tests mostly are concerned with ensuring the right page flow, and the right behavior for various controls.  There's some validation of the produced objects as well.  But I don't think this requires we maintain static fixtures.  If we could test the important behavior and validate the JSON objects the UI produces against the schema and the API programatically we'd still catch API definition changes since we've a typed API library.